app/views/editions/index.html.erb

```diff
@@ -16,6 +17,11 @@
           <%= link_to "See edition", edition %>
         </td>
         <td>
+        <% if edition.previously_published_edition %>
```

Comment:
Perhaps it's an anti-pattern to just hide this.

What about disabling it, and having some popup text explaining why it's not visible?